bundler/src/main/java/com/f2prateek/bundler/FragmentBundleBuilder.java

```diff
@@ -0,0 +1,419 @@
+package com.f2prateek.bundler;
+
+import android.app.Fragment;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.util.SparseArray;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+
+/**
+ * A bundle builder for fragment arguments.
+ *
+ * @param <F> The fragment type you're bundling to.
+ */
+@SuppressWarnings("unchecked")
+class FragmentBundleBuilder<F extends Fragment> extends Bundler {
```

Comment:
In the first version, I had it wrap the `Bundler` but then you have pretty much all the Bundler code repeated here, it's a lot of duplication, plus this class feels like it is just a specialization of the Bundler that also knows how to pass that info into a Fragment, so I think it makes more sense to extend it.